lib/linguist/presenters/loc_presenter.rb

```diff
@@ -0,0 +1,18 @@
+module Linguist
+  # This class handles the output of each of the
+  # command options
+  module Presenters
+    class Loc
+      def initialize(repo, io = $stdout)
+        @repo = repo
+        @io = io
+      end
+
+      def present
+        @repo.languages.each do |lang, lang_loc| 
```

Comment:
I'm looking for this feature as well, but the proposed implementation is incorrect.
The value in `lang_loc` isn't actually the number of lines but the number of characters.